docs/data/charts/hooks/use-dataset.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Charts - useDataset
+productId: x-charts
+---
+
+# useDataset
+
+<p class="description">Access the dataset used to populate series and axes data.</p>
+
+The `useDataset` hook provides access to the dataset array that was passed to the chart. This is useful when you need to access the raw data for custom components or calculations.
```

Comment:
Should we mention that what users should use if they want to access the `data` property?